docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -144,12 +166,13 @@ Here is some pseudocode for deriving the PAF and RR of "lack of access to CPAP"
   p_CPAP_CEmONC = 0.393
   RR_CPAP = 0.53
 
-  p_RDS_w = ... # solve system of equations from previous section
-  p_RDS_wo = ...
+  p_RDS_w_CPAP = ... # solve system of equations from previous section
+  p_RDS_wo_CPAP = # TODO: fill this in explicitly
 
   RR_no_CPAP = 1 / RR_CPAP
-  # p_RDS_w = (1 - PAF_no_CPAP) * p_RDS
+  # p_RDS_w_CPAP = (1 - PAF_no_CPAP) * p_RDS
   PAF_no_CPAP = 1 - p_RDS_w / p_RDS # solved equation in previous line for PAF
+  PAF_no_CPAP = 1 - 1 / (RR_no_CPAP * p_RDS_wo_CPAP + p_RDS_w_CPAP)
```

Comment:
This isn't right, is it? The coverage of CPAP doesn't seem to be entering in anywhere here...

Reply:
It is part of the calculation of p_RDS_wo_CPAP, e.g. if coverage of CPAP is zero, then p_RDS_wo_CPAP = p_RDS.

Reply:
But this equation for the PAF of no-CPAP does not appear to have any term in the right hand side that includes the coverage. It seems like it should be 
```suggestion
  PAF_no_CPAP = 1 - 1 / (RR_no_CPAP * p_no_CPAP + p_CPAP)
```

Reply:
And I don't understand how it relates to the previous two lines -- is it an alternative?

Reply:
I think this is an attempt at the alternative equation that Nathaniel proposed, and that I made the centerpiece of the exposition for antibiotics.